experiments/standalone/Soil/richards_runoff.jl

```diff
@@ -54,14 +71,103 @@ runoff_model = ClimaLand.Soil.Runoff.TOPMODELRunoff{FT}(;
     f_max = f_max,
     R_sb = R_sb,
 )
-vg_α = ClimaCore.Fields.ones(subsurface_space) .* FT(0.2)
-hydrology_cm = map(vg_α) do (α)
-    FT = typeof(α)
-    ClimaLand.Soil.vanGenuchten{FT}(; α = α, n = α + FT(2))
+soil_params_artifact_path = "/groups/esm/ClimaArtifacts/artifacts/soil_params_Gupta2020_2022"# @clima_artifact("soil_params_Gupta2020_2022")
+function mask_vg(var, value)
+    if var < 1e-8
+        return value
+    else
+        return var
+    end
 end
-θ_r = ClimaCore.Fields.zeros(subsurface_space)
-ν = ClimaCore.Fields.zeros(subsurface_space) .+ FT(0.5)
-K_sat = ClimaCore.Fields.zeros(subsurface_space) .+ FT(1e-6)
+vg_α = SpaceVaryingInput(
+    joinpath(
+        soil_params_artifact_path,
+        "vGalpha_map_gupta_etal2020_2.5x2.5x4.nc",
+    ),
+    "α",
+    subsurface_space;
+    regridder_type = :InterpolationsRegridder,
+    regridder_kwargs = (;
+        extrapolation_bc = (
+            Interpolations.Periodic(),
+            Interpolations.Flat(),
+            Interpolations.Flat(),
+        ),
+    ),
+)
+vg_α .= mask_vg.(vg_α, 1e-3)
+function mask_vg_n(var, value)
+    if var < 1
+        return value
+    else
+        return var
+    end
+end
+vg_n = SpaceVaryingInput(
+    joinpath(soil_params_artifact_path, "vGn_map_gupta_etal2020_2.5x2.5x4.nc"),
+    "n",
+    subsurface_space;
+    regridder_type = :InterpolationsRegridder,
+    regridder_kwargs = (;
+        extrapolation_bc = (
+            Interpolations.Periodic(),
+            Interpolations.Flat(),
+            Interpolations.Flat(),
+        ),
+    ),
+)
+vg_n .= mask_vg_n.(vg_n, 1.001)
+vg_fields_to_hcm_field(α::FT, n::FT) where {FT} =
+    ClimaLand.Soil.vanGenuchten{FT}(; @NamedTuple{α::FT, n::FT}((α, n))...)
+hydrology_cm = vg_fields_to_hcm_field.(vg_α, vg_n)
+
+θ_r = SpaceVaryingInput(
+    joinpath(
+        soil_params_artifact_path,
+        "residual_map_gupta_etal2020_2.5x2.5x4.nc",
+    ),
+    "θ_r",
+    subsurface_space;
+    regridder_type = :InterpolationsRegridder,
+    regridder_kwargs = (;
+        extrapolation_bc = (
+            Interpolations.Periodic(),
+            Interpolations.Flat(),
+            Interpolations.Flat(),
+        ),
+    ),
+)
+
+ν = SpaceVaryingInput(
+    joinpath(
+        soil_params_artifact_path,
+        "porosity_map_gupta_etal2020_2.5x2.5x4.nc",
+    ),
+    "ν",
+    subsurface_space;
+    regridder_type = :InterpolationsRegridder,
+    regridder_kwargs = (;
+        extrapolation_bc = (
+            Interpolations.Periodic(),
+            Interpolations.Flat(),
+            Interpolations.Flat(),
+        ),
+    ),
+)
+ν .= mask_vg.(ν, 1.0)
+K_sat = SpaceVaryingInput(
+    joinpath(soil_params_artifact_path, "ksat_map_gupta_etal2020_2.5x2.5x4.nc"),
+    "Ksat",
+    subsurface_space;
+    regridder_type = :InterpolationsRegridder,
+    regridder_kwargs = (;
+        extrapolation_bc = (
+            Interpolations.Periodic(),
+            Interpolations.Flat(),
+            Interpolations.Flat(),
```

Comment:
Since you are using this multiple times, you could put in a different variable and just pass it.